recipes/flask-rdf/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "flask_rdf" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6848d4e53fa83117555803cb7d52cdea1a2fafd793898f59d19bfc64cf27db8b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - python-mimeparse
```

Comment:
```suggestion
    - python-mimeparse 0.1.4
```

Reply:
updating python-mimeparse version to 0.1.4 results in build failure, as the version is not available in conda-forge. Hence allowing latest python-mimeparse version 

Thanks,
Ryan D'Mello